daq_lib.py

```diff
@@ -754,13 +755,13 @@ def checkC2C_X(x,fovx): # this is to make sure the user doesn't make too much of
   return 1
   
 
-def center_on_click(x,y,fovx,fovy,source="screen",maglevel=0,jog=0): #maglevel=0 means lowmag, high fov, #1 = himag with digizoom option, 
+def center_on_click(x,y,fovx,fovy,source="screen",maglevel=0,jog=0,unitscaling=1,axisX="x",axisY="y"): #maglevel=0 means lowmag, high fov, #1 = himag with digizoom option, 
```

Comment:
I believe the comment was AMX/FMX, it also looks like the maglevel variable is not being utilized. The axis values are meant to allow them to be swapped based on the position of the camera. If the camera is looking down at the sample from above, you can replace the Y axis movement with the focus. I think it may make more sense to offer configurations to select instead, to eliminate the need to give an explanation about the switching logic. Simply 
 offering choices like "Above" or "Below" and setting the configuration properly.

Reply:
Beamlines have multiple frames of reference - relative to the synchrotron, relative to the beam, etc. so being clear what the name is about is important.

To me, X and Y are arbitrary directions as when a 2-d plot is drawn on a surface. I think what you mean is that the X and Y axes are the horizontal and vertical directions as seen in the sample view camera. If you can change the names to reflect that better and add more text about what they mean, this would help the code make more sense.